### PR TITLE
Dynamic top bar navigation preview

### PR DESCRIPTION
**Pull request by gustavlrsn**
# Description

Adds a "Dynamic top bar" preview feature with an alternative top bar navigation with a bread crumb pattern.

# Screenshots

### CODE REVIEW

**Comment by gustavlrsn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #9526** <a href="https://app.graphite.dev/github/pr/opencollective/opencollective-frontend/9526" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opencollective/opencollective-frontend/9526?utm_source=stack-comment).

File: components/dashboard/DashboardSection.tsx [hunk 1]
```diff
@@ -90,15 +90,15 @@ const DashboardSection = ({ account, isLoading, section, subpath }) => {
   const SettingsComponent = SETTINGS_COMPONENTS[section];
   if (SettingsComponent) {
     return (
-      <div className="max-w-screen-md">
+      <div className=" max-w-screen-md">
```
Comment (kewitz) [inline components/dashboard/DashboardSection.tsx:93]: ?

**Review by kewitz — APPROVED** [review]
That looks good, nice work! I missed the correct cursor in a few places and pointed it out.

File: components/dashboard/DashboardSection.tsx (hunk 1 above)
Comment (gustavlrsn) [inline components/dashboard/DashboardSection.tsx:93]:
```suggestion
      <div className="max-w-screen-md">
```

File: components/dashboard/DashboardSection.tsx [hunk 2]
```diff
@@ -90,15 +90,15 @@ const DashboardSection = ({ account, isLoading, section, subpath }) => {
...
         <SettingsComponent account={account} subpath={subpath} />
       </div>
     );
   }
 
   if (values(LEGACY_SETTINGS_SECTIONS).includes(section)) {
     return (
-      <div className="max-w-screen-md">
+      <div className=" max-w-screen-md">
```
Comment (gustavlrsn) [inline components/dashboard/DashboardSection.tsx:101]:
```suggestion
      <div className="max-w-screen-md">
```

File: components/dashboard/DashboardSection.tsx (hunk 1 above)
Comment (gustavlrsn) [reply components/dashboard/DashboardSection.tsx]: Leftover, fixed that now

**Comment by gustavlrsn** [comment]
@kewitz thank you, I've added the cursors, good catch!

**Pull request merged**